chapters/introduction.tex

```diff
@@ -95,12 +93,11 @@ \section{Some Definitions}\doublelabel{some-definitions}
 
 The semantic specification should be read together with the Modelica
 grammar. Non-normative text, i.e., examples and comments, are enclosed
-  in {[} {]}; comments are set in italics. Additional terms are explained
+in {[}\ldots{]} and set in italics. Additional terms are explained
 in the glossary in \autoref{glossary}. Some important terms are:
-
```

Comment:
This empty line must be kept (or similarly) - otherwise it seems the table ends up mashed together with the text in the pdf.

Reply:
Good to know!  I hadn't realized that `tabular` was different from the various item environments that take care of breaking the current line, and where there is a difference between staying in the same paragraph and having a paragraph break before the environment.

Fixed.